gradle/dependency-locks/embulkPluginRuntime.lockfile

```diff
@@ -1,7 +1,24 @@
 # This is a Gradle generated file for dependency locking.
 # Manual edits can break the build and are not advised.
 # This file is expected to be part of source control.
+com.fasterxml.jackson.core:jackson-annotations:2.6.7
+com.fasterxml.jackson.core:jackson-core:2.6.7
+com.fasterxml.jackson.core:jackson-databind:2.6.7
+com.fasterxml.jackson.datatype:jackson-datatype-guava:2.6.7
+com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.6.7
+com.google.guava:guava:18.0
+com.ibm.icu:icu4j:54.1.1
 commons-codec:commons-codec:1.10
+commons-io:commons-io:2.7
 commons-logging:commons-logging:1.2
+javax.validation:validation-api:1.1.0.Final
 org.apache.httpcomponents:httpclient:4.5.6
 org.apache.httpcomponents:httpcore:4.4.10
+org.embulk:embulk-util-config:0.3.1
+org.embulk:embulk-util-file:0.1.3
+org.embulk:embulk-util-guess:0.2.0
+org.embulk:embulk-util-json:0.1.0
+org.embulk:embulk-util-retryhelper:0.8.1
+org.embulk:embulk-util-text:0.1.1
+org.json:json:20200518
+org.msgpack:msgpack-core:0.8.11
```

Comment:
`org.msgpack.msgpack-core` should not be included here. Please try finding where it comes from, and `exclude` it.